dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -366,6 +368,21 @@ public class ConnectorArguments extends DefaultArguments {
           .ofType(String.class)
           .describedAs("principal/host");
 
+  private final OptionSpec<String> optionHiveSaslQop =
```

Comment:
I would suggest using a different naming convention for Hive-specific flags: `-Dhiveql.hive-sasl-qop`. For an example, please see the `-Dteradata.metadata.tablesizev.max-rows=value` flag:
https://github.com/google/dwh-migration-tools/blob/9162ae5823a8e46468a5758d571a4976a204f180/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataMetadataConnector.java#L66-L67

As for other Hive-specific flags that do not follow this convention, we can migrate them later.

Reply:
Correction: `-Dhiveql.sasl-qop`. It doesn't make sense to repeat the `hive` word twice in the option name.